ably/options.go

```diff
@@ -450,37 +474,62 @@ func (opts *clientOptions) activePort() (port int, isDefault bool) {
 	return
 }
 
-func (opts *clientOptions) getRestHost() string {
+// endpointFqdn handles an endpoint that uses a hostname, which may be an IPv4
+// address, IPv6 address or localhost
+func endpointFqdn(endpoint string) bool {
```

Comment:
```suggestion
func endpointFQDN(endpoint string) bool {
```

See https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
Also, given this returns a bool, I suggest:

```suggestion
func isEndpointFQDN(endpoint string) bool {
```